schema_source/cloudformation-docs.json

```diff
@@ -9999,70 +10142,6 @@
       "Type": "The type of this environment tier\\.  \nValid values:  \n+ For *Web server tier* \u2013 `Standard` \n+ For *Worker tier* \u2013 `SQS/HTTP` \n*Required*: No  \n*Type*: String  \n*Update requires*: [Replacement](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-replacement)",
       "Version": "The version of this environment tier\\. When you don't set a value to it, Elastic Beanstalk uses the latest compatible worker tier version\\.  \nThis member is deprecated\\. Any specific version that you set may become out of date\\. We recommend leaving it unspecified\\.\n*Required*: No  \n*Type*: String  \n*Update requires*: [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt)"
     },
-    "AWS::ElasticLoadBalancing::LoadBalancer": {
```

Comment:
Removed in https://github.com/awsdocs/aws-cloudformation-user-guide/commit/a4c118131592451bd34e8d3c839e5e0c9c66fab8, reached out to them.